chia/seeder/start_crawler.py

```diff
@@ -51,10 +54,13 @@ def service_kwargs_for_full_node_crawler(
 
 
 def main():
+    # TODO: refactor to avoid the double load
+    full_config = load_config(DEFAULT_ROOT_PATH, "config.yaml")
     config = load_config_cli(DEFAULT_ROOT_PATH, "config.yaml", "seeder")
+    full_config[SERVICE_NAME] = config
```

Comment:
```suggestion
    # TODO: refactor to avoid the double load
    config = load_config(DEFAULT_ROOT_PATH, "config.yaml")
    service_config = load_config_cli(DEFAULT_ROOT_PATH, "config.yaml", "seeder")
    config[SERVICE_NAME] = service_config
```

Reply:
Added as a `Draft for:` reason.